### PR TITLE
Add StyleLint

### PR DESCRIPTION
**Pull request by HorstFrank**
### StyleLint

- https://stylelint.io/user-guide/get-started
  - `npm install --save-dev stylelint stylelint-config-standard`
  - `echo '{\n"extends": "stylelint-config-standard"\n}' > .stylelintrc.json`
  - check the installation with: `npx stylelint "**/*.css"`
  - Add a style script `"slint": "stylelint '**/*.css' ",`
 
##### StyleLint with Prettier
- https://github.com/prettier/stylelint-config-prettier
  - `npm install --save-dev stylelint-config-prettier`
  - Then, append stylelint-config-prettier to the extends array in your `.stylelintrc.\*` file. Make sure to put it last, so it will override other configs. ` { "extends": [ "stylelint-config-standard", "stylelint-config-prettier" ] }`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint .",
+    "slint": "stylelint '**/*.css' ",
```
Comment (lmachens) [inline package.json]:
```suggestion
    "slint": "stylelint '**/*.css'",
```

File: README.md [hunk 2]
```diff
@@ -107,3 +107,14 @@ Check out our [Next.js deployment documentation](https://nextjs.org/docs/deploym
...
     - AND add a Setting Part after the rules ... think of the comma after teh rules Object!!!
       - `"settings": { "react": { "version": "detect" } }`
+
+#### StyleLint
+
+- https://stylelint.io/user-guide/get-started
+  - `npm install --save-dev stylelint stylelint-config-standard`
+  - `echo '{\n"extends": "stylelint-config-standard"\n}' > .stylelintrc.json`
+  - check the installation with: `npx stylelint "**/*.css"`
+  - Add a style script `"slint": "stylelint '**/*.css' ",`
```
Comment (lmachens) [inline README.md:117]:
```suggestion
  - Add a style script `"slint": "stylelint '**/*.css'",`
```

**Review by lmachens — APPROVED** [review]
good job! only unnecessary whitespace detected 👀

**Review by phlgr — APPROVED** [review]
🚀

File: package.json (hunk 1 above)
Comment (HorstFrank) [reply package.json]: Thanks!

**Review by fre-ben — APPROVED** [review]
Great job as always, Boris! 👍

**Review by christophvogel — APPROVED** [review]
GJ 👍

File: package.json [hunk 3]
```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint .",
+    "slint": "stylelint '**/*.css'",
```
Comment (m-wiemers) [inline package.json]:
```suggestion
    "slint": "stylelint \"**/*.css\"",
```
can help

**Review by m-wiemers — CHANGES_REQUESTED** [review]
gj 🚀

**Comment by DNaunin** [comment]
GJ! 🚀

**Review by SilviaBarcellini — APPROVED** [review]
Great! 👍

**Pull request merged**